extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/remote_inference_client/text_generation.py

```diff
@@ -301,14 +307,24 @@ async def run_inference(
 
         # if stream enabled in runtime options and config, then stream. Otherwise don't stream.
         stream = True  # Default value
-        if options is not None and options.stream is not None:
-            stream = options.stream
+        if (
+            sanitized_options is not None
+            and sanitized_options.stream is not None
+        ):
+            stream = sanitized_options.stream
```

Comment:
does this need to use santized options? curious to hear the reasoning behind using sanitized here vs keeping this unchanged

Reply:
Technically no, but it's possible someone might add code in the future and do something with the options (which could involve logging), so IMO it's safer to consistently use the sanitized_options throughout